File: 3.cameraandmatrices/webgl/source/js/app/CCAppEngine.js
/*-----------------------------------------------------------
 * 2c - Cross Platform 3D Application Framework
 *-----------------------------------------------------------
 * Copyright © 2011 - 2011 France Telecom
 * This software is distributed under the Apache 2.0 license.
 * http://www.apache.org/licenses/LICENSE-2.0.html
 *-----------------------------------------------------------
 * File Name   : CCAppEngine.js
 * Description : 2c start point
 *
 * Created     : 25/04/10
 * Author(s)   : Ashraf Samy Hegab
 *-----------------------------------------------------------
 */
 
CCEngine.prototype.start = function()
{
	this.addScene( new SceneSample() );
}

File: 3.cameraandmatrices/mobile/App/Source/CCAppEngine.cpp
/*-----------------------------------------------------------
 * 2c - Cross Platform 3D Application Framework
 *-----------------------------------------------------------
 * Copyright © 2010 - 2011 France Telecom
 * This software is distributed under the Apache 2.0 license,
 * see the "license.txt" file for more details.
 *-----------------------------------------------------------
 * File Name   : CCAppEngine.cpp
 *-----------------------------------------------------------
 */

#include "CCDefines.h"
#include "Samples.h"


void CCAppEngine::start()
{
    addScene( new SceneSample() );
    
    //AddFlag( gEngine->renderFlags, collision_box );
}
